src/robusta/core/triggers/pod_crash_loop_trigger.py

```diff
@@ -1,29 +1,32 @@
 from ..discovery.top_service_resolver import TopServiceResolver
 from ...core.playbooks.base_trigger import TriggerEvent
-from ...integrations.kubernetes.autogenerated.triggers import PodUpdateTrigger, PodChangeEvent
+from ...integrations.kubernetes.autogenerated.triggers import (
+    PodUpdateTrigger,
+    PodChangeEvent,
+)
 from ...integrations.kubernetes.base_triggers import K8sTriggerEvent
 from ...utils.rate_limiter import RateLimiter
 
 
 class PodCrashLoopTrigger(PodUpdateTrigger):
-    rate_limit: int = 3600
-    restart_reason: str = None
-    restart_count: int = 2
-
     """
-    :var rate_limit: Limit firing to once every `rate_limit` seconds 
+    :var rate_limit: Limit firing to once every `rate_limit` seconds
```

Comment:
Moving this docstring to the top of the class is the only real change here. Everything else is just black autoformating.